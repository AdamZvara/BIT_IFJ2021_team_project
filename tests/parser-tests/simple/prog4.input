require "ifj21"

global foo : function(integer)

function bar(n : integer, s : string)
	local str : string = s
	write(str,"\n")
	if 10 > 5 then
		write("Bigger")
	else
		write("smaller")
	end
	foo(n)
end

function foo(n : integer)
	write(n,"\n")
	n = n + 2
	write(n)
	write("\n")
end
bar(5,"ahoj")
