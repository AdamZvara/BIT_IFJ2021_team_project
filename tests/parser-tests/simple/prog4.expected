ahoj
Bigger5
7
